core/common/src/rpc/IModelTileRpcInterface.ts

```diff
@@ -35,7 +61,7 @@ export abstract class IModelTileRpcInterface extends RpcInterface {
   }
 
   /** @internal */
-  public async requestTileTreeProps(_tokenProps: IModelRpcProps, _id: string): Promise<TileTreeProps> { return this.forward(arguments); }
+  public async requestTileTreeProps(_tokenProps: IModelRpcProps, _id: string): Promise<IModelTileTreeProps> { return this.forward(arguments); }

```

Comment:
Is this going to require backend and frontend to upgrade in unison?

Reply:
See [TileProps.ts](https://github.com/imodeljs/imodeljs/pull/163/files?w=1#diff-7a5382b7172e75b797895684a9b8b76b8d41dec1636e03c603aca400ce0ec8cd). The same properties will be returned, plus one extra (`geometryGuid`) used only by interactive editing applications (aka nobody who isn't upgrading backend and frontend in unison).

Reply:
I did see that, but you removed some properties from TileTreeProps and moved them to IModelTileTreeProps.  I just wondered if this would cause some issues during the change over.

Reply:
I understand. It won't. See [Tree::ToJson()](https://dev.azure.com/bentleycs/iModelTechnologies/_git/imodel02/commit/c372af2bc54e07302637aa8efc26ea6291d449df?refName=refs%2Fheads%2Fmaster&path=%2FiModelCore%2FDgnPlatform%2FDgnCore%2FTile.cpp) - still returns the same stuff (plus `geometryGuid`) - all that's changed is the compile-time TypeScript interface.

Reply:
OK.